release/mk-cfgman-docs.sh

```diff
@@ -42,12 +42,12 @@ for v in ${VERS3} "$@" ; do
 done
 
 # Dynamically detect which v4+ releases exist right now and grok them all.
-VERS4=`ls -1t ${webpath}/Versions/ | grep -v "CVS|sig.dyn"`
+VERS4=`ls -1t ${webpath}/content/Versions/ | grep -v "CVS|sig.dyn"`
```

Comment:
FWIW, AFAICT, there are non-version files in that directory that are ignored by this "filter non-versions" `grep`:

```
langpack
index.dyn
makeweb.sh
authors.sh
```

They will be skipped due to the `test -d` below, but so is `sig.dyn` file present in this `grep` regex. Thus, even if we ignore the fact that files like `auhtors.sh` are not "versions", there is an inconsistency here.